### PR TITLE
Add options to beautify button

### PR DESCRIPTION
**Pull request by Bqleine**
This PR introduces two new options for the beautify button :
 - **Center vertically when beautifying** will add a newline at the top of your string if it is one or two lines long.
 - **Add newlines if there are less than 4 lines** will add newlines at the end of your string if it is less than four lines long.

It also modifies the algorithm to stop if from removing already existing newlines (requested by Rafaeeel).

New layout:
2022-11-20-211226_411x528_scrot

### CODE REVIEW

File: beautify.js [hunk 1]
```diff
@@ -50,14 +57,36 @@ const beautifyText = async () => {
...
+const initBeautify = async () => {
     const button = document.querySelector('#beautify-button');
     button.addEventListener('click', beautifyText);
+
+    const CVInput = document.querySelector('#center-vertically');
+    const NFInput = document.querySelector('#newlines-fill');
+
+    const { centerVertically, newlinesFill } = await browser.storage.local.get(['centerVertically', 'newlinesFill']);
+
+    CVInput.checked = centerVertically === undefined ? true : centerVertically;
```
Comment (ImRodry) [inline beautify.js:82]: Don't default it to true
```suggestion
    CVInput.checked = centerVertically;
```

Comment (ImRodry) [inline beautify.js:89]: Don't default it to true
```suggestion
    CVInput.checked = centerVertically;
```

**Comment by Bqleine** [comment]
Don't you dare touch my handcrafted, beautiful commit with your unbased opinions

**Comment by Tobost06** [comment]
Really cool!

**Comment by Brythzz** [comment]
Works great, thank you!

**Pull request merged**